contribs/gmf/src/directives/editfeature.js

```diff
@@ -0,0 +1,324 @@
+goog.provide('gmf.EditfeatureController');
+goog.provide('gmf.editfeatureDirective');
+
+goog.require('gmf');
+/** @suppress {extraRequire} */
+goog.require('gmf.featurestyleDirective');
+goog.require('ngeo.ToolActivate');
+goog.require('ngeo.ToolActivateMgr');
+/** @suppress {extraRequire} */
+goog.require('ngeo.btngroupDirective');
+/** @suppress {extraRequire} */
+goog.require('ngeo.drawfeatureDirective');
+goog.require('ngeo.interaction.Modify');
+goog.require('ol.Collection');
+
+
+/**
+ * Directive used to create, modify and delete vector features on a map with
+ * the addition of changing their style.
+ * Example:
+ *
+ *     <gmf-editfeature
+ *         gmf-editfeature-active="ctrl.editFeatureActive">
+ *         gmf-editfeature-map="::ctrl.map">
+ *     </gmf-editfeature>
+ *
+ * @htmlAttribute {boolean} gmf-editfeature-active Whether the directive is
+ *     active or not.
+ * @htmlAttribute {ol.Map} gmf-editfeature-map The map.
+ * @return {angular.Directive} The directive specs.
+ * @ngInject
+ * @ngdoc directive
+ * @ngname gmfEditfeature
+ */
+gmf.editfeatureDirective = function() {
```

Comment:
If one day we want to edit a real feature (from WFS), edit its attributes for example. Would it be in same directive ?
Is the name `editFeature` is well chosen ?